common/dynamicconfig/setting.go

```diff
@@ -0,0 +1,49 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
```

Comment:
```suggestion
// Copyright (c) 2024 Temporal Technologies Inc.  All rights reserved.
//
```

Reply:
Do we have a policy on this? It is kind of derived from the old code in some sense so maybe it's not worth changing?